### PR TITLE
Add ETC fork names

### PR DESCRIPTION
**Pull request by fubuloubu**
Also added description about how to use the EVM versioning system

### Cute Animal Picture
fat monkey

### CODE REVIEW

File: tests/parser/functions/test_get_hash.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def foo4() -> bytes32:
     return get_extcodehash(self.a)
     """
 
-    if evm_version == "byzantium":
+    if evm_version == "byzantium" or evm_version == "atlantis":
```
Comment (iamdefinitelyahuman) [inline tests/parser/functions/test_get_hash.py]:
```suggestion
    if evm_version in ("byzantium", "atlantis"):
```

I think a tuple is a cleaner approach, especially if other named rulesets might be added in the future.

File: tests/compiler/test_opcodes.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def test_version_check(evm_version):
     assert opcodes.version_check(begin=evm_version)
     assert opcodes.version_check(end=evm_version)
     assert opcodes.version_check(begin=evm_version, end=evm_version)
-    if evm_version != "byzantium":
+    if evm_version != "byzantium" and evm_version != "atlantis":
```
Comment (jacqueswww) [inline tests/compiler/test_opcodes.py]: I think we should perhaps consider enumerating these values ?

```
class EVMVersion:
     ATLANTIS = "atlantis"
```


Also I believe `evm_version in {"byzantium", "atlantis"}` is easier to read than two and statements.

File: tests/parser/functions/test_get_hash.py (hunk 1 above)
Comment (jacqueswww) [reply tests/parser/functions/test_get_hash.py]: Oh wow lol; just made the same suggestion but with a set instead :laughing:

File: tests/parser/functions/test_bitwise.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def _shift(x: uint256, y: int128) -> uint256:
 @pytest.mark.parametrize('evm_version', list(EVM_VERSIONS))
 def test_bitwise_opcodes(evm_version):
     opcodes = compile_code(code, ['opcodes'], evm_version=evm_version)['opcodes']
-    if evm_version == "byzantium":
+    if evm_version == "byzantium" or evm_version == "atlantis":
```
Comment (fubuloubu) [inline tests/parser/functions/test_bitwise.py]:
```suggestion
    if evm_version == "byzantium" or evm_version == "atlantis":
    if evm_version in ("byzantium", "atlantis"):
```

File: tests/compiler/test_opcodes.py [hunk 4]
```diff
@@ -53,7 +53,7 @@ def test_get_opcodes(evm_version):
     else:
         assert "CHAINID" not in op
         assert op['SLOAD'][-1] == 200
-    if evm_version == "byzantium":
+    if evm_version == "byzantium" or evm_version == "atlantis":
```
Comment (fubuloubu) [inline tests/compiler/test_opcodes.py]:
```suggestion
    if evm_version in ("byzantium", "atlantis"):
```

File: tests/compiler/test_opcodes.py (hunk 2 above)
Comment (fubuloubu) [reply tests/compiler/test_opcodes.py]:
```suggestion
    if evm_version in ("byzantium", "atlantis"):
```

File: tests/compiler/test_opcodes.py [hunk 5]
```diff
@@ -39,7 +39,7 @@ def test_version_check(evm_version):
     assert opcodes.version_check(begin=evm_version)
     assert opcodes.version_check(end=evm_version)
     assert opcodes.version_check(begin=evm_version, end=evm_version)
-    if evm_version != "byzantium":
+    if evm_version in ("byzantium", "atlantis"):
```
Comment (fubuloubu) [inline tests/compiler/test_opcodes.py]:
```suggestion
    if evm_version not in ("byzantium", "atlantis"):
```

**Pull request merged**